cockroachcloud/managing-cmek.md

```diff
@@ -72,16 +73,76 @@ Activate CMEK with a call to the clusters CMEK endpoint, using the cloud-specifi
 
 See the [API specification](../api/cloud/v1.html#operation/CockroachCloud_EnableCMEK).
 
-{% include_cached copy-clipboard.html %}
-```shell
-CLUSTER_ID= #{ your cluster ID }
-API_KEY= #{ your API key }
-curl --request POST \
-  --url https://cockroachlabs.cloud/api/v1/clusters/${CLUSTER_ID}/cmek \
-  --header "Authorization: Bearer ${API_KEY}" \
-  --header 'content-type: application/json' \
-  --data "@cmek_config.json"
-```
+1. Create a new file named `cmek_config.json`. This file will contain a JSON array of `region_spec` objects, each of which includes the name of a {{ site.data.products.db }} region and a `key_spec` that is specific to the target KMS platform and specifies the URI of the CMEK key and the credential that authorizes {{ site.data.products.db }} to encrypt and decrypt using the key.
```

Comment:
We should probably change "credential" to "authorization object" as that's what's being mentioned above.

Reply:
Ah yeah. Consider "principal that is authorized", too. We call it `auth_principal`

Reply:
Went with the second suggestion as it reads more cleanly.